packages/dicom-codec/src/utils/logger.js

```diff
@@ -26,8 +26,8 @@ const logger = (() => {
     }
   }
 
-  function setVerbose() {
-    verbose = true;
+  function setVerbose(newVerbose = true) {
```

Comment:
Allows turning on/off logs later on, which helps quite a bit for debugging specific items.

Reply:
is it logger for build or that runs on every decoding?

Reply:
It is on every decoding, and I wanted to be able to turn it on/off for test purposes after load.  Previously, it got turned on at load time if it was on at all, and couldn't be turned off.